src/nacl/bindings/utils.py

```diff
@@ -39,3 +39,59 @@ def sodium_memcmp(inp1, inp2):
     eqC = lib.sodium_memcmp(buf1, buf2, ln) == 0
 
     return eqL and eqC
+
+
+def sodium_increment(inp):
+    """
+    Increment the value of a byte-sequence interpreted
+    as a little-endian big integer
+
+    :param inp: input bytes buffer
+    :type inp: bytes
+    :return: integer value of ``inp`` incremented by one
```

Comment:
This documentation doesn't seem right. The integer interpretation has incremented by one, but this function returns a byte string.